utils/Makefile.am

```diff
@@ -86,4 +82,8 @@ EXTRA_DIST = \
 	thapi_metadata_tracepoints.tp \
 	command.rb \
 	meta_parameters.rb \
-	optparse_thapi.rb
+	optparse_thapi.rb \
+	thapi_log_to_bt_source_component.rb \
+	thapi_model_remove_packet_context_fields.rb \
+	thapi_text_pretty_set_ts.rb \
```

Comment:
This file is never used correct? Just useful when manually editing  files?

Reply:
correct.